src/main.ts

```diff
@@ -439,23 +439,18 @@ ipcMain.handle(
   }
 )
 
-ipcMain.handle('har:open', async (_, fileName: string): Promise<HarFile> => {
-  console.info('har:open event received')
```

Comment:
I noticed we have some events have this log and others do not, any specific reasons why we might not want them anymore ? 🤔

Reply:
I should've discussed this with the team first, but I don't think these bring a lot of value, while making the code a bit harder to maintain (it's a bit of s stretch, I know):
- You need to remember to add it when adding a new handler
- You need to remember to update it if you rename the channel

Also these logs aren't written to the app logs, so they are only useful when debugging something locally, in which case wouldn't it be easier to add temporary console.logs in the local branch?

Reply:
I agree they are more for debugging rather than info
Depending on what you are doing might be useful to see all the events your operations are doing to track what's happening, if you have to add them back manually might be a slog 🤔 

> You need to remember to update it if you rename the channel
hopefully that's not something that happens often, specially post GA 😅
I also see how they can be error-prone, annoying, the ideal scenario would be for electron to an event when it arrives automatically 🤔

Reply:
Yes, I wish there was a simple way to add a middleware for logging all this automatically

Reply:
There's also [this doc](https://www.electronjs.org/docs/latest/tutorial/debugging-main-process) on debugging in the main process